src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String TEXT_SPACER = "    ";
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        printLineSpacer();
+        System.out.println(TEXT_SPACER + "Hey, what's up!\n" + TEXT_SPACER + "What can I help you with today?");
+        printLineSpacer();
+
+        Task[] userLists = new Task[100];
+        Scanner scanner = new Scanner(System.in);
+        String userLineInput = "";
+        int numOfTasks = 0;
+
+        do {
+            userLineInput = scanner.nextLine();
+            String[] splitInputs = userLineInput.split(" ");
+            String userCommand = splitInputs[0];
+            printLineSpacer();
+            if (!userCommand.equals(("bye"))) {
+                switch (userCommand) {
+                case "list":
+                    printList(userLists, numOfTasks);
+                    break;
+                case "done":
+                    completeTask(userLists, splitInputs);
+                    break;
+                case "event":
+                case "deadline":
+                case "todo":
+                    storeTasks(userLists, userLineInput, numOfTasks, userCommand);
+                    numOfTasks++;
+                    break;
+                default:
+                    System.out.println(TEXT_SPACER + "Invalid Input!");
+                    printLineSpacer();
+                    break;
+                }
+            }
+        } while(!userLineInput.equals("bye"));
+
+        System.out.println(TEXT_SPACER + "Aight. See you soon mate.");
+        printLineSpacer();
+    }
+
+    private static void printList(Task[] userLists, int numOfTasks) {
+        if (numOfTasks == 0) {
+            System.out.println(TEXT_SPACER + "List is empty!");
+        } else {
+            System.out.println(TEXT_SPACER + "Here's your list of tasks:");
+            for (int i = 0; i < numOfTasks; i++) {
+                System.out.println("    " + (i + 1) + "." + userLists[i].toString());
+            }
+        }
+        printLineSpacer();
+    }
+
+    private static void completeTask(Task[] userLists, String[] splitInput) {
+        int chosenEntry;
+        chosenEntry = Integer.parseInt(splitInput[1]) - 1;
+        userLists[chosenEntry].markAsDone();
+        System.out.println(TEXT_SPACER + "Good job on completing a task!");
+        System.out.println(TEXT_SPACER + userLists[chosenEntry].toString());
+        printLineSpacer();
+    }
+
+    private static void storeTasks(Task[] userLists, String userLineInput, int numOfTasks, String userCommand) {
```

Comment:
I like the usage of different methods to address the addition of different types of tasks, makes it easier to edit if the structure and definitions of different objects change :)